readme.md

```diff
@@ -4,6 +4,28 @@ submit your own, if they make me laugh I'll merge them.
 
 ---
 
+```
+[Please Enter New Password]
+
+fortnight
+
+[Error: Password is Two Week]
+```
+
+---
+
+**Q:** How many Prolog programmers does it take to change a lightbulb?
+
+**A:** Yes.
```

Comment:
You'll have to explain this one to me 😄

Reply:
[Prolog](https://en.wikipedia.org/wiki/Prolog) is a logic programming language. It works in such a way that you establish facts and rules, and then you are able to ask whether certain relationships can be established between atoms. This makes certain types of problems far easier to solve than if you were to implement a solution in _e.g._ JavaScript. Conversely, it makes solving other kinds of computations relatively hard. In the book _Seven Languages in Seven Weeks_, Prolog was compared to Dustin Hoffman's character in _Rainman_ — working in Prolog is a bit like communicating with an autistic savant.

The answers are typically `yes` or `no`.

If you have familiarity with Prolog, this joke is quite funny.